Umbraco-Cloud/Databases/Backups/index.md

```diff
@@ -0,0 +1,62 @@
+## Backups
+It's possible to create a backup of a Cloud database. There are at least two ways of accomplishing this:
+- Use SQL Server Management Studio
```

Comment:
Would be cool if these two where links to the sections in the article.
Example:
`[Use SQL Server Management Studio](#Backup-with-SQL-server-management-studio)` <- will link to the first section